task-launcher/cypress/e2e/hearts_and_flowers.cy.js

```
@@ -1,73 +1,39 @@
-const PORT = Cypress.env('DEV_PORT') || 8080;
-const hearts_and_flowers_url = `http://localhost:${PORT}/?task=hearts-and-flowers`;
+const hearts_and_flowers_url = 'http://localhost:8080/?task=hearts-and-flowers';
 
 // keep track of game phase (true means it has started)
-var heart_phase = false;
-var flower_phase = false;
-var mixed_practice = false;
-var final_instructions = false; // instructions before final mixed test phase
-var mixed_test = false;
-let lastProgressAt = Date.now();
-let instructionClicks = 0;
-let answersClicked = 0;
-const markProgress = (label) => {
-  cy.task('progress', label);
-  cy.log(`[progress] ${label}`);
-  lastProgressAt = Date.now();
-};
+let heart_phase = false;
+let flower_phase = false;
+let mixed_practice = false;
+let final_instructions = false; // instructions before final mixed test phase
+let mixed_test = false;
 
 describe('test hearts and flowers', () => {
   it('visits hearts and flowers and plays game', () => {
+    cy.intercept({ resourceType: /xhr|fetch/ }, { log: false });
```

What does this do? Silently fails network requests?